server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -23,39 +23,41 @@
 import io.seata.core.constants.ConfigurationKeys;
 
 /**
- * The type parameter parser
+ * The type Parameter parser.
  *
  * @author xingfudeshi@gmail.com
  * @date 2019/05/30
  */
 public class ParameterParser {
     private static final String PROGRAM_NAME = "sh seata-server.sh(for linux and mac) or cmd seata-server.bat(for windows)";
     private static final int SERVER_DEFAULT_PORT = 8091;
-    private static final String SERVER_DEFAULT_BIND_IP = "0.0.0.0";
     private static final String SERVER_DEFAULT_STORE_MODE = "file";
     private static final int SERVER_DEFAULT_NODE = 1;
+    /**
+     * The constant CONFIG.
+     */
     protected static final Configuration CONFIG = ConfigurationFactory.getInstance();
 
     @Parameter(names = "--help", help = true)
     private boolean help;
-    @Parameter(names = {"--host", "-h"}, description = "The host to bind.", order = 1)
-    private String host = SERVER_DEFAULT_BIND_IP;
+    @Parameter(names = {"--host", "-h"}, description = "The ip to register to registry center.", order = 1)
+    private String host;
     @Parameter(names = {"--port", "-p"}, description = "The port to listen.", order = 2)
     private int port = SERVER_DEFAULT_PORT;
     @Parameter(names = {"--storeMode", "-m"}, description = "log store mode : file、db", order = 3)
     private String storeMode = CONFIG.getConfig(ConfigurationKeys.STORE_MODE, SERVER_DEFAULT_STORE_MODE);
     @Parameter(names = {"--serverNode", "-n"}, description = "server node id, such as 1, 2, 3. default is 1", order = 4)
     private int serverNode = SERVER_DEFAULT_NODE;
 
+    /**
+     * Instantiates a new Parameter parser.
+     *
+     * @param args the args
```

Comment:
Abnormal description

Reply:
The first args is the name of the parameter, and the second args is the explanation of the parameter.